pkgs/development/python-modules/strict-rfc3339/default.nix

```
@@ -0,0 +1,18 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "strict-rfc3339";
+  version = "0.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5cad17bedfc3af57b399db0fed32771f18fc54bbd917e85546088607ac5e1277";
+  };
+
```

Sorry, realized there's no tests in tarball so we should disable the `checkPhase`
```suggestion
doCheck = false;
```